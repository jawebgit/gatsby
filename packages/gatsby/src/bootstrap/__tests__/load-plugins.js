const loadPlugins = require(`../load-plugins`)

describe(`Load plugins`, () => {
  it(`load plugins for a site`, async () => {
    const plugins = await loadPlugins({ plugins: [] })

    expect(plugins).toMatchSnapshot()
  })
})
