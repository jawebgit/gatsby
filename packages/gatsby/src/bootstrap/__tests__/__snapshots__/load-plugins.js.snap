// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Load plugins load plugins for a site 1`] = `
Array [
  Object {
    "name": "component-page-creator",
    "pluginOptions": Object {
      "plugins": Array [],
    },
    "resolve": "/Users/kylemathews/programs/gatsby/packages/gatsby/src/bootstrap/internal-plugins/component-page-creator",
    "version": "1.0.0",
  },
  Object {
    "name": "internal-data-bridge",
    "pluginOptions": Object {
      "plugins": Array [],
    },
    "resolve": "/Users/kylemathews/programs/gatsby/packages/gatsby/src/bootstrap/internal-plugins/internal-data-bridge",
    "version": "1.0.0",
  },
  Object {
    "name": "defaultSitePlugin",
    "pluginOptions": Object {
      "plugins": Array [],
    },
    "resolve": "/Users/kylemathews/programs/gatsby",
    "version": "n/a",
  },
]
`;
